core/message-tags-3.3.md

```diff
@@ -179,6 +201,19 @@ A message sent by a client with the `+example-client-tag` client-only tag:
 
 ---
 
+A message with the same tags sent with both the normal and JSON encodings for comparison:
+
+    @tag=value;+client-tag=value\swith\sspaces PRIVMSG #channel :Message
+    @{"tag":"value","+client-tag":"value\swith\spaces"} PRIVMSG #channel :Message
```

Comment:
If you're going to specify JSON, then the data ends when the JSON structure ends.

Escaping the " " is not necessary.